react/introduction/setting_up_a_react_environment.md

```diff
@@ -79,9 +79,19 @@ Provided everything has gone according to plan, head over to `localhost:5173`, w
 
 Congratulations! You've created your first React app.
 
-<div class="lesson-note lesson-note--tip" markdown=1>
+To link your local project directory to a GitHub repo, create a new **empty** repo on GitHub then follow the instructions in the new repo's page to connect it to your local project directory.
 
-As you might've noticed by now, you can replace `my-first-react-app` with the name of your project.
+<div class="lesson-note lesson-note--tip" markdown="1">
+
+#### Using an existing repo
+
+Alternatively, if you created a GitHub repo already and cloned it, you can `cd` into your cloned repo then run the above Vite command, using `.` as the project name:
+
+```bash
+npm create vite@latest . -- --template react
+```
+
+This will tell Vite to use the current directory for the project, instead of creating a new directory with the given project name. This cloned directory will already be initialized as a git repo and connected to the right remote.
```

Comment:
I'm debating whether we have a need for this. At this point in the curriculum, learner has sufficient knowledge of git (as they've gone through the foundations and the advanced git lessons). I'm assuming they can figure this out themselves?

Reply:
This is less of a thing with Git knowledge and more not realising that the project name will be the directory that Vite scaffolds a project in, creating the directory if it doesn't exist (hence people not knowing that `.` will scaffold into the current directory). It's an extremely common question in the react-help channel about whether they *have* to create a new dir with the vite command then hook it up to a git repo, or whether they can clone a git repo then create a vite project within it.

Reply:
In that case, I'd rather include a mention that `.` scaffolds into the current directory than adding git instructions. What do you think?

Reply:
This PR includes both:
1. One of the most common questions is how to commit and push the new Vite project given it's not a Git repo yet (since the original command creates a new `my-react-apo` folder). While the help is simple, given it's the very first time in the curriculum (the JS pathway for sure) that people will have done things this way, I think it's sensible to complete the instructions with initialising the Vite project as a git repo and connecting the remote (line 82).
2. The `.` bit is an extra and answers the very common question and knowledge gap that `.` as the project name uses the current directory.

While the first point (git instructions) is the most important IMHO (the instructions feel noticeably incomplete otherwise), the second point addresses the more common question asked in the server.

Reply:
eh, I still feel it's a bit tooo handholdy, but... you've conviced me

Reply:
The primary justification is how often those same questions need to be answered in the help channels, which indicates to me that even though we'd want people to figure it out themselves (which many do), many do actually get confused and feel like the instructions are incomplete.